src/front/app/features/challenge/challenge.reducer.js

```diff
@@ -12,8 +14,12 @@ const reducer = (state = initialState, action = {}) => {
         return upsertChallenge(action.challenge, state);
```

Comment:
@borys-ihnatyev, [what is upsert](https://www.google.com.ua/search?q=what+is+upsert&oq=what+is+upsert&aqs=chrome..69i57.3759j0j4&sourceid=chrome&ie=UTF-8)